AOAT-1

JaimeGG64 opened the pull request:
This will include webpack on the project. To start make sure you have yarn, NPM, and node.js.
Once you have all of the requirements run.
``` bash
$ yarn install
```
Then run
``` bash
$ yarn dev
```
or
``` bash
$ yarn watch
```
To compile the CSS.
To Check if it is compiling correctly just open the ` studentworkgd.html` and make simple CSS changes to the SCSS files located `~/src/sass`

JaimeGG64 committed:
install a couple of package to use for the AOAT site

JaimeGG64 committed:
delete the 'version didnt see the light' and bootstrap folders

JaimeGG64 committed:
rearranged files and delete old files

JaimeGG64 committed:
delete some more unnessary files and package.json file

JaimeGG64 committed:
got scss to mostly work with webpack and node-sass

JaimeGG64 committed:
create some test with node-sass

BishoyAbdelmalik reviewed (commented):
did you delete the 'version didn't see the light' and bootstrap folders or not???
I am so confused
other than that looks good

JaimeGG64 commented:
I deleted `/version didn't see the light` and moved the `bootstrap` folder because webpack should handle bundling our source code. If you feel that its good merge to the `dev` branch.
